ui/lib/core/addon/templates/components/replication-action-generate-token.hbs

```diff
@@ -14,14 +14,12 @@
   </div>
 
   <div class="action-block-action">
-    <button
-      type="button"
-      class="button is-tertiary"
-      onclick={{action (mut this.isModalActive) true}}
+    <Hds::Button
```

Comment:
Even though the class says `tertiary` this is how the original looked:
<img width="170" alt="Screenshot 2023-11-13 at 5 41 34 PM" src="https://github.com/hashicorp/vault/assets/68122737/75e705cd-5fd4-4eb8-baf8-2b5f795bc0b8">